scripts/patch-release.sh

```diff
@@ -1,16 +1,17 @@
 #!/bin/bash
 
+cd ..
```

Comment:
Why call this patch-release? I think you can use this script for building and deploying a minor (or even a major) release as well? Though for those we should perhaps skip the template news item/release note thing as they are typically done manually, in advance.

Reply:
That's what it was designed for. But yeah, it will work for minor/major releases if you merge to master first. I'll rename the script back again.